types/redux-form/lib/actions.d.ts

```diff
@@ -18,8 +18,8 @@ export declare function arraySplice(form: string, field: string, index: number,
 export declare function arraySwap(form: string, field: string, indexA: number, indexB: number): FormAction;
 export declare function arrayUnshift(form: string, field: string, value: any): FormAction;
 export declare function autofill(form: string, field: string, value: any): FormAction;
-export declare function blur(form: string, field: string, value: any): FormAction;
-export declare function change(form: string, field: string, value: any): FormAction;
+export declare function blur(form: string, field: string, value: any, touch: boolean): FormAction;
```

Comment:
I believe the major version should change because of the `touch` parameter that was missing and now is required, but not sure how to make it consistent with the version of `redux-form` itself

Reply:
I think the reality is breaking changes happen all the time. It's hard to always keep these definitions aligned with both the library and typescript changes in a way that doesn't break end user code. We also don't really have a good way to provide a major version for breaking changes since we have to match the library version. I for one have accepted that this and just deal with them as they come.

FWIW this is an area of the code that users shouldn't be touching much anyways.

Reply:
Agreed